plugin/semantic.py

```diff
@@ -0,0 +1,232 @@
+from .code_actions import CodeActionOrCommand
+from .core.protocol import Diagnostic
+from .core.protocol import Request
+from .core.registry import LspTextCommand
+from .core.registry import windows
+from .core.sessions import SessionBufferProtocol
+from .core.typing import List, Optional, Any, Dict, Tuple, Sequence, cast
+from .core.views import text_document_identifier
+
+import sublime
+import sublime_plugin
+import time
+
+SUBLIME_WORD_MASK = 515
+
+NON_MODIFIER_SCOPES = {
+            'variable'     :'variable.other.lsp',
+            'parameter'    :'variable.parameter.lsp',
+            'function'     :'variable.function.lsp',
+            'method'       :'variable.function.lsp',
+            'property'     :'variable.other.member.lsp',
+            'class'        :'entity.name.class.lsp',
```

Comment:
I think this may lead to problems, because `entity.name` scopes should only used for definitions. See my Java example in a comment above. But I'm not sure what would be the best scope name here. Do you maybe have a screenshot where this scope is marked in e.g. magenta, so we can see where exactly this "class" token type is used by a server?